capstone-ui/src/components/ProfileCard/ProfileCard.jsx

```diff
@@ -41,13 +41,15 @@ export default function ProfileCard({
   return (
     <div className="profilecard-component">
       <div className="profile-picture-wrapper">
-        {appProfile &&
-          appProfile.imageURL &&
+        {appProfile != null &&
+          appProfile != undefined &&
+          appProfile.imageURL != undefined &&
```

Comment:
appProfile != null  should also cover appProfile != undefined (whereas appProfile !== null does not). you can double check, but I think you don't need appProfile != undefined

https://stackoverflow.com/questions/16672743/javascript-null-check